### PR TITLE
Add keywords for make-mode

### PR DESCRIPTION
**Pull request by twlz0ne**

### CODE REVIEW

File: company-keywords.el [hunk 1]
```diff
@@ -317,3 +337,4 @@
 
 (provide 'company-keywords)
 ;;; company-keywords.el ends here
+
```
Comment (dgutov) [inline company-keywords.el:340]: Please drop the addition of the last empty line.

File: company-keywords.el [hunk 2]
```diff
@@ -297,6 +297,26 @@
...
   "Alist mapping major-modes to sorted keywords for `company-keywords'.")
 
+(with-eval-after-load 'make-mode
+  (require 'seq)
+  (mapc
+   (lambda (mode-stmnts)
+     (setf (alist-get (car mode-stmnts) company-keywords-alist)
+           (seq-uniq
+            (append
+             makefile-statements
```
Comment (dgutov) [inline company-keywords.el:307]: `makefile-automake-statements` and others seem to already include `makefile-statements`.

So I think it should be dropped from here and instead replace the `nil` values below.

File: company-keywords.el [hunk 3]
```diff
@@ -297,6 +297,26 @@
...
+             makefile-special-targets-list
+             (mapcan #'identity
```
Comment (dgutov) [inline company-keywords.el:309]: Might also want to pass it through `cl-delete-duplicates`.

File: company-keywords.el (hunk 1 above)
Comment (twlz0ne) [reply company-keywords.el:340]: Sorry for this.

File: company-keywords.el (hunk 2 above)
Comment (twlz0ne) [reply company-keywords.el:307]: I'll fix it next push.

File: company-keywords.el (hunk 3 above)
Comment (twlz0ne) [reply company-keywords.el:309]: I think `seq-uniq` (line 305) did the same thing.

Comment (dgutov) [reply company-keywords.el:308]: Not sure how I missed that. Thanks.

**Comment by dgutov** [comment]
Could you try fixing the CI error as well? Maybe `(eval-when-compile (require 'make-mode))` will help.

**Comment by dgutov** [comment]
And next, how do you feel about signing the papers assigning the copyright to FSF?

These two PRs have reached the allowed limit (and even exceeded it a little), so the above will be required for any more contributions.

**Comment by dgutov** [comment]
Thanks!

**Comment by dgutov** [comment]
Consider doing the copyright assignment, though.

**Pull request merged**